compose/ui/ui/src/uikitMain/kotlin/androidx/compose/ui/scene/ComposeSceneLayerView.uikit.kt

```diff
@@ -0,0 +1,117 @@
+/*
+ * Copyright 2024 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package androidx.compose.ui.scene
+
+import androidx.compose.ui.input.pointer.PointerEventType
+import androidx.compose.ui.window.centroidLocationInView
+import kotlinx.cinterop.CValue
+import kotlinx.cinterop.readValue
+import platform.CoreGraphics.CGPoint
+import platform.CoreGraphics.CGRectZero
+import platform.UIKit.UIEvent
+import platform.UIKit.UITouch
+import platform.UIKit.UIView
+
+/**
+ * A backing ComposeSceneLayer view for each Compose scene layer. Its task is to
+ * handle events that start outside the bounds of the layer content
+ *
+ * @param isInsideInteractionBounds a function that returns true if the given point is inside the
+ * interactable content part of the layer.
+ * @param isInterceptingOutsideEvents a function that returns true if the layer should intercept
+ * events that start outside the bounds of the layer content or should let them pass through.
+ */
+internal class ComposeSceneLayerView(
```

Comment:
I have doubts about `ComposeSceneLayerView` and `ComposeSceneLayersView` as well as `UIKitComposeSceneLayer` and `ComposeSceneLayer` naming.

Let's rename it into `UIKitComposeSceneLayerView` and think about better naming for `ComposeSceneLayers` and `ComposeSceneLayersView` (Maybe `ComposeLayersHolder(View)` or something better.

Reply:
Renamed